packages/redux/src/products/reducer/recentlyViewedProducts.ts

```diff
@@ -55,9 +55,10 @@ const result = (
         remote: action.payload,
         pagination: omit(action.payload, 'entries') as RecentlyViewedProducts,
         computed: uniqBy(
-          sortBy(
+          orderBy(
             [...action.payload.entries, ...computed],
             entry => new Date(entry.lastVisitDate),
+            'desc',
```

Comment:
A suggestion will be the order to be a param on `result()` arguments.
i.e.:` result(state, action, order = 'desc')`

Reply:
The reducer is invoked by redux directly, we cannot pass other arguments to it directly.
We could have this in the `getRecentlyViewedProducts` selector but I think the descending order is what makes sense for all cases. If there is a need to have it ordered differently, the consumer will have to do it.